Developers/Writing_Bears.rst

```diff
@@ -276,3 +276,98 @@ parameter in ``HiddenResult`` should be the instance of the Bear that yields
 this result (in this case ``self``), and second argument should be the content
 we want to transfer between the Bears. Here we use a list of strings as content
 but it can be any object.
+
+More Configuration Options
+--------------------------
+coala provides settings to further configure your bear according to your needs.
+Here are all the settings you can use:
+
+``LANGUAGES``: To indicate which languages your bear supports,
+just give it the ``LANGUAGES`` value which should be a set of string(s):
```

Comment:
bad exprimation, you dont give "it" the languages value. languages is a variable, you need to give it a set of strings as values :)